e2e/playwright/pagination/src/tests/favorites.ts

```diff
@@ -0,0 +1,110 @@
+      expect(await favoritePage.pagination.isPreviousEnabled()).toBe(false);
+      expect(await favoritePage.pagination.isNextEnabled()).toBe(true);
+    });
+
+    test('[C280114] Items per page values', async ({ favoritePage }) => {
+      await favoritePage.pagination.openMaxItemsMenu();
+      expect(await (await favoritePage.pagination.getNthItem(1)).innerText()).toBe('25');
+      expect(await (await favoritePage.pagination.getNthItem(2)).innerText()).toBe('50');
+      expect(await (await favoritePage.pagination.getNthItem(3)).innerText()).toBe('100');
+      await favoritePage.pagination.closeMenu();
+    });
+
+    test('[C280115] current page menu items', async ({ favoritePage }) => {
+      await favoritePage.pagination.openMaxItemsMenu();
+      await favoritePage.pagination.clickMenuItem('25');
+      expect(await favoritePage.pagination.getMaxItems()).toContain('25');
+      expect(await favoritePage.pagination.getTotalPages()).toContain('of 3');
+      expect(await favoritePage.pagination.getItemsCount()).toBe(3);
+      await favoritePage.pagination.closeMenu();
+
+      await favoritePage.pagination.openMaxItemsMenu();
+      await favoritePage.pagination.clickMenuItem('50');
+      expect(await favoritePage.pagination.getMaxItems()).toContain('50');
+      expect(await favoritePage.pagination.getTotalPages()).toContain('of 2');
+
+      await favoritePage.pagination.closeMenu();
+
+      await favoritePage.pagination.openMaxItemsMenu();
+      await favoritePage.pagination.clickMenuItem('100');
+      expect(await favoritePage.pagination.getMaxItems()).toContain('100');
+      expect(await favoritePage.pagination.getTotalPages()).toContain('of 1');
+
+      await favoritePage.pagination.resetToDefaultPageSize();
+    });
+
+    test('[C280116] change the current page from menu', async ({ favoritePage }) => {
+      await favoritePage.pagination.clickOnNextPage();
+      expect(await favoritePage.pagination.getRange()).toContain('Showing 26-50 of 51');
+      expect(await favoritePage.pagination.getCurrentPage()).toContain('Page 2');
+      expect(await favoritePage.pagination.isPreviousEnabled()).toBe(true);
+      expect(await favoritePage.pagination.isNextEnabled()).toBe(true);
+      await favoritePage.pagination.resetToDefaultPageSize();
+    });
+
+    test('[C280119] navigate to next and previous pages', async ({ favoritePage }) => {
+      await favoritePage.pagination.openMaxItemsMenu();
+      await favoritePage.pagination.clickMenuItem('25');
+      expect(await favoritePage.pagination.getMaxItems()).toContain('25');
+      await favoritePage.pagination.clickOnNextPage();
+      expect(await favoritePage.pagination.getRange()).toContain('Showing 26-50 of 51');
+
+      await favoritePage.pagination.clickOnPreviousPage();
+      expect(await favoritePage.pagination.getRange()).toContain('Showing 1-25 of 51');
+    });
+
+    test('[C280117] Previous button is disabled on first page', async ({ favoritePage }) => {
```

Comment:
here test steps are already getting verified in test [C280113] Pagination control default values.
We can have different steps or remove the test.

Reply:
Hi, I think we can remove the test as it is redundant but I am not sure if this will have to be removed from test rail as well? 
I checked, and these tests are present on test rail. What do you suggest?

Reply:
we can mark as deprecated and can move to delete folder